src/PrestaShopBundle/EventListener/AssignSmartyVariablesListener.php

```diff
@@ -0,0 +1,151 @@
+            'help_link' => 'TODO',
+            'img_dir' => $this->imageDirectory,
+            'lite_display' => 'TODO',
+            'baseAdminUrl' => $this->baseUri . basename(_PS_ADMIN_DIR_) . '/',
+            'base_url' => $this->globalVariables->getBaseUrl(),
+            'toggle_navigation_url' => $this->legacyContext->getAdminLink('AdminEmployees', true, [
+                'action' => 'toggleMenu',
+            ]),
+            'collapse_menu' => $this->globalVariables->isMenuCollapsed(),
+            'enableSidebar' => 'TODO',
+            'title' => 'TODO',
+            'meta_title' => 'TODO',
+            'table' => 'TODO',
+            'tabs' => 'TODO',
+            'default_tab_link' => $this->globalVariables->getDefaultTabLink(),
+            'current_tab_level' => $this->menuBuilder->getCurrentTabLevel(),
+            'lang_is_rtl' => $this->languageContext->isRTL(),
+            'full_language_code' => $this->languageContext->getLanguageCode(),
+            'country_iso_code' => $this->countryContext->getIsoCode(),
+            'currentIndex' => $this->legacyControllerContext->currentIndex,
+            'default_language' => (int) $this->configuration->get('PS_LANG_DEFAULT'),
+            'js_router_metadata' => $this->globalVariables->getJsRouterMetadata(),
+            'locked_to_all_shop_context' => $this->globalVariables->isLockedToAllShopContext(),
+            'token' => $this->legacyControllerContext->token,
+            'employee' => $this->legacyContext->getContext()->employee,
+            'is_multishop' => $this->globalVariables->isMultiShop(),
+            'shop_name' => $this->shopContext->getName(),
+            'shop' => $this->legacyContext->getContext()->shop,
+            'shop_group' => $this->legacyContext->getContext()->shop->getGroup(),
+            'iso' => $this->globalVariables->getIsoUser(),
+            'class_name' => $this->legacyControllerContext->className,
+            'version' => $this->globalVariables->getVersion(),
+            'link' => $this->legacyContext->getContext()->link,
+            'controller_name' => $this->legacyControllerContext->controller_name,
+            'css_files' => $this->legacyControllerContext->css_files,
+            'js_files' => $this->legacyControllerContext->js_files,
+            'js_def' => $this->legacyControllerContext->js_files,
+            'errors' => $this->legacyControllerContext->errors,
+            'warnings' => $this->legacyControllerContext->warnings,
+            'informations' => $this->legacyControllerContext->informations,
+            'confirmations' => $this->legacyControllerContext->confirmations,
+            'login_link' => $this->legacyContext->getAdminLink('AdminLogin'),
+            'logout_link' => $this->legacyContext->getAdminLink('AdminLogin', true, ['logout' => 1]),
+            'round_mode' => $this->configuration->get('PS_PRICE_ROUND_MODE'),
+            'help_box' => $this->configuration->get('PS_HELPBOX'),
```

Comment:
Don't know what it's used for, no traces in any smarty templates so e can remove it